app/src/main/java/org/gnucash/android/model/Price.java

```diff
@@ -78,6 +95,7 @@ public void setType(String type) {
     }
 
     public long getValueNum() {
+        reduce();
```

Comment:
This is causing the valueNum of the price to be halved when it is retrieved. 
What is the purpose of this `reduce()` call here?